tests/Auth0.ManagementApi.IntegrationTests/JobsTest.cs

```diff
@@ -125,5 +125,26 @@ public async Task Can_import_users()
                 importUsers.Connection.Should().Be(_auth0Connection.Name);
             }
         }
+
+        [Fact]
+        public async Task Can_export_users()
+        {
+            var request = new UsersExportsJobRequest
+            {
+                ConnectionId = _auth0Connection.Id,
+                Format = UsersExportsJobFormat.JSON,
+                Limit = 1,
+                Fields = new System.Collections.Generic.List<UsersExportsJobField> { new UsersExportsJobField { Name = "email" } }
+            };
+
+            var exportUsers = await _apiClient.Jobs.ExportUsersAsync(request);
+            exportUsers.Should().NotBeNull();
+            exportUsers.Id.Should().NotBeNull();
+            exportUsers.Type.Should().Be("users_export");
+            exportUsers.Status.Should().Be("pending");
+            exportUsers.CreatedAt.Should().BeCloseTo(DateTime.UtcNow, TimeSpan.FromMinutes(5));
```

Comment:
Shouldn't the test also be checking that the `fields` array contains `email`?

Reply:
Oh, nice catch, thanks!

Reply:
I had to do some research for this, but my findings are the following:

- Different Jobs endpoint can return different response type, all sharing a generic Job, enriched with some of the properties sent as part of the request.
- Both Auth0-Java and Auth0.NET ignore endpoint specific fields and only return the generic Job type and its field.

At first, I thought we should add these as you commented, so we can test them. But after more consideration I am not sure we need them, given that if you send a field to an API that replies with an HTTP 200, I assume the fields are accepted and I do not need them as part of the response.